jobs/config.json

```diff
@@ -415,6 +415,141 @@
       "UNKNOWN"
     ]
   }, 
+  "ci-kubernetes-e2e-gce-cos-beta-k8s-current": {
```

Comment:
Not sure how is the job name used in Prow. If this cannot be changed, we probably want to use `ci-kubernetes-e2e-gce-cosbeta-k8scurrent`.